platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/tipset/TipsetMetrics.java

```diff
@@ -40,15 +46,37 @@ public class TipsetMetrics {
                     + "node's events as other parents.");
     private final RunningAverageMetric bullyScoreMetric;
 
+    private final Map<NodeId, SpeedometerMetric> parentMetrics = new HashMap<>();
```

Comment:
If you're calling the other metric `pityParentMetric`, perhaps this should be called `tipsetParentMetric`? Otherwise the implication seems to be that this would be cycled each time *any* other parent is used

Reply:
done